scenes.ts

```diff
@@ -258,6 +260,7 @@ export function initializeSceneEndpoints(state: State) {
     content: SceneContent,
     outgoing_url: t.union([t.string, t.undefined]),
     text: t.string,
+    publish: t.boolean,
```

Comment:
For consistency, I feel like it might be better to name this `published`.

Reply:
Also, I think that it would be better to make this field optional, defaulting to `true`. Adding a requirement field is break of the JSON API, and while we don't exactly have a ton of deployed code out there that's using this API, I'd like to stay in the habit of maintaining compatibility when feasible.